libcxx/test/benchmarks/spec.gen.py

```diff
@@ -28,6 +28,9 @@
 link_flags = (test_dir / 'link_flags.subs').open().read().strip()
 spec_dir = pathlib.Path((test_dir / 'spec_dir.subs').open().read().strip())
 
+# Remove -Wall, -Wextra and -Werror from the flags since we don't care about enabling all warnings inside SPEC
+compile_flags = compile_flags.replace('-Wall', '').replace('-Wextra', '').replace('-Werror', '')
```

Comment:
I don't think we ever care about the warnings, do we? In that case we might as well just pass `-w`.